zerver/migrations/0576_namedusergroup_creator_namedusergroup_date_created.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 5.0.7 on 2024-08-24 07:01
+
+import django.db.models.deletion
+import django.utils.timezone
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0575_alter_directmessagegroup_group_size"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="namedusergroup",
+            name="creator",
+            field=models.ForeignKey(
+                db_column="creator",
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="+",
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
+        migrations.AddField(
+            model_name="namedusergroup",
+            name="date_created",
+            field=models.DateTimeField(default=django.utils.timezone.now, null=True),
+        ),
+    ]
```

Comment:
See `zerver/migrations/0511_stream_creator.py` and 63f593620740343cb2b51c0e1c1336b60816cc45 for what you need to know to add a `RunPython` rule to backfill` creator` and `date_created` where we have the audit log data required to do so.

Reply:
If you're satisfied with the PR and would like to merge it, I can backfill the changes in a new PR on Saturday, if that works for you

Reply:
So the backfill migration should always be a separate migration file; you can see we do that constantly if you read past migrations. The reason is transactions; it's a lot nicer if the operations to add a new column are in a small transaction that can't get rolled back due to a bug in the backfill code. Gonna fix this before merging.